content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -902,16 +773,13 @@ and therefore won't affect [validation](#validation) in the API server.
 
 ### Additional printer columns
 
-Starting with Kubernetes 1.11, kubectl uses server-side printing. The server decides which
-columns are shown by the `kubectl get` command. You can customize these columns using a
+The kubectl tool reliese on server-side output formatting. Your cluster decides which
```

Comment:
Spelling: relies
Instead of `Your cluster`, substitute `The API server`?

Reply:
```suggestion
The kubectl tool relies on server-side output formatting. Your cluster's API server decides which
```